build.gradle

```diff
@@ -70,16 +71,14 @@ ext {
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
 }
 
-group = ext.groupId
```

Comment:
I'm now accessing the groudId directly via rootProject.ext.groupId in the publish.gradle file as I don't see how this additional step helps with anything. And it's not documented why it's there.